Assignment 2/180501105/occurences.py

```diff
@@ -0,0 +1,5 @@
+from collections import Counter as c
+s=input("Enter a string:")
+a=c(s)
+print(a)
```

Comment:
Print the output in the required format.

Reply:
Changes made